python/lsst/ci/prepare.py

```diff
@@ -49,53 +58,31 @@ def __str__(self):
         return message
 
 
-class Product:
-    """Class representing an EUPS product to be built"""
-    def __init__(self, name, sha1, version, dependencies):
-        self.name = name
-        self.sha1 = sha1
-        self.version = version
-        self.dependencies = dependencies
-
-    def flat_dependencies(self):
-        """Return a flat list of dependencies for the product.
-
-            Returns:
-                list of `Product`s.
-        """
-        res = set(self.dependencies)
-
-        for dep in self.dependencies:
-            res.update(dep.flat_dependencies())
-
-        return res
-
-
 class Manifest:
     """A representation of topologically ordered list of EUPS products to be built
 
-       :ivar products: topologically sorted list of `Product`s
+       :ivar product_index: topologically sorted list of `Product`s
```

Comment:
This is pre-existing but what's with ":ivar"? Our standard is to use:
```
Parameters
------------
    product_index : type (which may be optional if you use type annotation?)
        description of product_index...
```
as per https://developer.lsst.io/python/numpydoc.html#py-docstring-class-structure

Please fix all the doc strings in this file if you are willing. I see "Args" instead of "Parameters" and the underline is missing.